docs/cloud-config/index.md

```diff
@@ -87,3 +87,12 @@ rancher:
         - 8.8.4.4
 
 ```
+
+#### Set Hostname
+
+You can set the hostname of the host using cloud-config. The example below shows how to configure it.
+
+```yaml
+hostname: myhost
+```
```

Comment:
Can you make notes about how if you set net: host, then the hostname will automatically be "rancher"? We probably need a better explanation than what I have put in the issue, but I don't understand why it gets overwritten.

https://github.com/rancherio/os/issues/217